### PR TITLE
SB-188 pet weight easter egg

### PR DESCRIPTION
**Pull request by katrinajaneczko**
Every 5 minutes, the unit of measurement for weight for the pet changes. The weights are all accurate (I checked/calculated).

If you want to test it, change line 364 of PetDisplay: `}, 5 * 60 * 1000); // change weight every 5 minutes`. For changing every 10 seconds, make it `10 * 1000` instead.

### CODE REVIEW

**Comment by katrinajaneczko** [comment]
Screenshot_20230414_131719_Gallery

File: pocs-capstone/frontend/src/components/PetDisplay/PetDisplay.js [hunk 1]
```diff
@@ -36,6 +36,36 @@ const HAPPY = 'H'
 const TASK_URL = "/tasks/"
 const USER_URL = "/user-data/"
 const TODAY = new Date()
+
+const WEIGHTS = [
+    '15 paperclips',
+    '3 nickels',
+    '5 pieces of bubblegum',
+    '4 mullberries',
+    '2/3 a medium radish',
```
Comment (harrisonfedor) [inline pocs-capstone/frontend/src/components/PetDisplay/PetDisplay.js:45]: ``` 2'3 of a medium radish ```

File: pocs-capstone/frontend/src/components/PetDisplay/PetDisplay.js [hunk 2]
```diff
@@ -36,6 +36,36 @@ const HAPPY = 'H'
...
+    '461 sequins',
+    '25 miniature toadstools',
+    '5 acorns',
+    '1 small egg',
+    '231.5 grains of rice',
+    '5 spoonfuls of cinnamon',
```
Comment (harrisonfedor) [inline pocs-capstone/frontend/src/components/PetDisplay/PetDisplay.js:51]: teaspoons? tablespoons? world's largest spoons? i don't know what my pet weighs!

File: pocs-capstone/frontend/src/components/PetDisplay/PetDisplay.js [hunk 3]
```diff
@@ -36,6 +36,36 @@ const HAPPY = 'H'
...
+    '12 jellybeans',
+    '1 strawberry',
+    '3 sheets of US letter sized printer paper',
+    '300 almond slivers',
+    '23/4 a slice of multigrain bread',
+    '1 big handful of cotton balls',
+    '4 clementine slices',
+    '150 wooden toothpicks',
+    '1 large thimble',
+    '3 #2 HB yellow Ticonderoga wooden pencils'
```
Comment (harrisonfedor) [inline pocs-capstone/frontend/src/components/PetDisplay/PetDisplay.js:65]: ``` 3 Ticonderoga #2 wooden pencils ```  maybe?

**Review by harrisonfedor — APPROVED** [review]
nice, clean solution and truly remarkable calculations. a fun addition to the app!

approved, but made a few (very important) comments for you to consider before merging

File: pocs-capstone/frontend/src/components/PetDisplay/PetDisplay.js (hunk 1 above)
Comment (katrinajaneczko) [reply pocs-capstone/frontend/src/components/PetDisplay/PetDisplay.js:45]: this does sound better ty

File: pocs-capstone/frontend/src/components/PetDisplay/PetDisplay.js (hunk 2 above)
Comment (katrinajaneczko) [reply pocs-capstone/frontend/src/components/PetDisplay/PetDisplay.js:51]: It's really 5.769 tsp of cinnamon but the pet's weight may fluctuate very slightly over the course of time.
I will make it 6 teaspoonfuls of cinnamon :D

File: pocs-capstone/frontend/src/components/PetDisplay/PetDisplay.js (hunk 3 above)
Comment (katrinajaneczko) [reply pocs-capstone/frontend/src/components/PetDisplay/PetDisplay.js:65]: lowkey I wanna go either "wooden pencils" or go the extremely overdescriptive route, but I already did that for the paper so I'm thinking I just make this wooden pencils

File: pocs-capstone/frontend/src/components/PetDisplay/PetDisplay.js (hunk 1 above)
Comment (katrinajaneczko) [reply pocs-capstone/frontend/src/components/PetDisplay/PetDisplay.js:45]: thanks to this I also noticed "23/4 a slice of multigrain bread" should have "of"

**Review by Alexander-Russakoff — APPROVED** [review]
Looking good!

**Pull request merged**